vod/media_format.h

```diff
@@ -14,7 +14,16 @@
 // constants
 #define MAX_CODEC_NAME_SIZE (64)
 #define MAX_FRAME_SIZE (10 * 1024 * 1024)
-#define MAX_TRACK_COUNT (1024)
+#ifdef NGX_VOD_MAX_TRACK_COUNT
+#if NGX_VOD_MAX_TRACK_COUNT % 64 != 0
+#error MAX_TRACK_COUNT must be a multiple of 64!
+#endif
+#define MAX_TRACK_COUNT (NGX_VOD_MAX_TRACK_COUNT)
+#else
+#define MAX_TRACK_COUNT (64)
+#endif
+// The parentheses in MAX_TRACK_COUNT are included in the stringification
+#define MAX_TRACK_INDEX_LEN (sizeof(ngx_value(MAX_TRACK_COUNT)) - 2 - 1)
```

Comment:
would prefer to put it inside the if/else, and use NGX_VOD_MAX_TRACK_COUNT/hardcoded value of 2, instead of doing -2